Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -72,6 +78,74 @@ def createAuthorizedSession(self, userLogin, userPassword):
         authorized_session.headers.update({ "Authorization":authorization_header })
         return authorized_session
 
+
+class Test_allScheduleControlTest(testCase):
+
+# # # # # # # # # # # # #
+# SCHEDULECONTROL TESTS #
+# # # # # # # # # # # # #
+
+#    Get the schedulecontrol object of the currently logged in user.
+#    Endpoint -- api/schedulecontrol
+#    Expected Status code -- 200 Ok
+#    Expected Content -- all schedule objects of the currently logged in user.
+
+    def test_get_all_schedulecontrol_objects_of_current_user(self):
```

Comment:
Is there a way to assert the information given by api/schedulecontrol of 360.student? And maybe specify what schedulecontrol object is. Is it the course schedule of the student?

Reply:
No, there is not a good way to asser the information from api/schedulecontrol for 360.studenttest.